src/test/java/jenkins/plugins/github/api/mock/MockUser.java

```diff
@@ -0,0 +1,63 @@
+package jenkins.plugins.github.api.mock;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+public class MockUser extends MockOwner<MockUser> {
```

Comment:
Wonder if we might use code generation from the published API definitions to create these classes?  Would reduce maintenance costs for adding support for new types @stephenc , no?
Possible also that Github has published a java client that already includes DTO types like this that we can use as a base.